common/lib/xmodule/xmodule/seq_module.py

```diff
@@ -248,19 +255,16 @@ def _hidden_content_student_view(self, context):
         display depending on whether staff is masquerading.
         """
         if not self._can_user_view_content():
-            subsection_format = (self.format or _("subsection")).lower()  # pylint: disable=no-member
-
-            # Translators: subsection_format refers to the assignment
-            # type of the subsection, such as Homework, Lab, Exam, etc.
-            banner_text = _(
-                "Because the due date has passed, "
-                "this {subsection_format} is hidden from the learner."
-            ).format(subsection_format=subsection_format)
+            if getattr(self, 'self_paced', False):
+                banner_text = _("Because the course has ended, this assignment is hidden from the learner.")
+                raise Exception("HAHAHAHAHA")
```

Comment:
Can we instead update or add a regresssion test to prove that we hit this point?